cookbooks/bcpc-hadoop/recipes/core_site.rb

```diff
@@ -56,6 +56,8 @@
 
      'hadoop.security.group.mapping.ldap.search.attr.group.name' =>
        'cn',
+     'hadoop.security.group.mapping.ldap.search.group.hierarchy.levels' =>
+       '1',
```

Comment:
As each additional hierarchy is quite expensive, and our ldap only has 1 extra level, I don't foresee us ever raising it higher, and if we do need to, we'd need to think about that as it will slow down every single hdfs request.  So I think it is safe to leave it here.